Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
"check some other object properties to show a different route." is typically the purpose of a Voter.